view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -25,7 +25,14 @@
                 </div>
             </div>
         </div>
-        <div data-bind="visible: isActive() && orderValueTooLow()" class="page messages">
+        <div data-bind="visible: !isAnyInstallmentOptionActive()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <div data-bind="i18n: 'There are no any installment options available'"></div>
```

Comment:
is that correct english?
_There are no any installment options available_

Reply:
`There are no instalment options available`